scripts/plugins/support-pssl.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#
+# Copyright (c) Citrix Systems 2017. All rights reserved.
+
+set -e
+
+# Open/Close port 6640 when pssl is used in ovs
+
+OP="$1"
+rule="RH-Firewall-1-INPUT -p tcp -m conntrack --ctstate NEW -m tcp --dport 6640 -j ACCEPT"
+
+case "${OP}" in
+    start)
+        if iptables -S | grep -e "${rule}" > /dev/null
+        then
+            echo "port 6640 is opened already"
+        else
+            iptables -A ${rule}
+            service iptables save
+        fi
+        ;;
+    stop)
+        iptables -D $rule
+        service iptables save
+        ;;
+    *)
+        echo $"Usage: $0 {start|stop}" 1>&2
```

Comment:
The string is using the Bash extquote feature `$"`, the echo above does not. I have no idea whether we really make use of extquote for internationalisation but both echos should be treated the same.
http://unix.stackexchange.com/questions/48106/what-does-it-mean-to-have-a-dollarsign-prefixed-string-in-a-script